src/main.cpp

```diff
@@ -6719,7 +6719,7 @@ bool ProcessMessage(CNode *pfrom, std::string strCommand, CDataStream &vRecv, in
     else if (strCommand == NetMsgType::XBLOCKTX && !fImporting && !fReindex && !IsInitialBlockDownload() &&
              IsThinBlocksEnabled())
     {
-        return CXThinBlockTx::HandleMessage(vRecv, pfrom);
+        return CXThinBlockTx::HandleMessage(vRecv, pfrom, strCommand);
```

Comment:
I needed to add it for HandleBlockMessage which now has to be called from within xblocktx.  I'd rather leave the functionality as intended for now and then modify in a separate PR.  We might be able to do away with strCommand entirely since we only need it for one reject message when we process the block.